src/lib/middleware/api-token-middleware.test.ts

```diff
@@ -228,6 +235,7 @@ test('should call next if apiTokenService throws x2', async () => {
         },
     };
 
+    // @ts-expect-error missing methods
```

Comment:
After properly defining the type at `{ apiTokenService }: any,` we get compiler errors here because we're just defining a single method

Reply:
Solved this in a more elegant way